spec.md

```diff
@@ -86,23 +86,23 @@ The configuration must satisfy $S_{\text{max}} \ge S_{\text{min}} \ge W > 0$.
 
 ## Definitions
 
-The "split index" $I(X)$, is either the smallest integer $i$ satisfying:
+The "split index" $I(X)$ of a sequence $X$ is either the smallest integer $i$ satisfying:
 
 - $i < |X|$ and
 - $S_{\text{max}} \ge i \ge S_{\text{min}}$ and
-- $H(\langle X_{i-W+1}, \dots, X_i \rangle) \mod 2^T = 0$
+- $H(\langle X_{\max(0, i-W)}, \dots, X_{i-1} \rangle) \mod 2^T = 0$
```

Comment:
The intention had been for `I(X)` to be the maximum index that is _included_ in the chunk, so this is correct as-is (but other things are wrong).  Some things below are perhaps cleaner to express if `I(X)` is the length of the chunk, so I'm fine changing this. But  in that case we also need to change the first condition to `i <= |X|`)